### DISCUSSION TITLE
Not able to see a step's logs while it is running

### DISCUSSION THREAD

**Discussion by MickaMx** [community/community#33052] [Actions]
Hello, this issue has been around for 2 years and no further information has been publish. Anyone who got the same issue ?

https://github.com/orgs/community/discussions/25450

**Comment by AmeenZahir21** [comment]
In the execution of a step that has been assigned a log file, it is not possible to see the logs that are associated with that step during its execution, so you will not be able to monitor them while they are being executed. [link redacted for safety]

**Reply by MickaMx** [reply]
So, if i remove the log file assigned to the step; i should be able te see logs as they run?

**Comment by bitdeft** [comment]
An issue was raised for this; details and further information can be seen here:

https://github.com/actions/runner/issues/886

@ethomson said that such feature requests should be raised in the discussion forum here. I am working with build debugging actively and can add a +1 to making this feature a priority, especially with steps that take a long time to complete, and provide vital information during their execution for debugging purposes.
